### PR TITLE
HDX-9848 currency endpoint

### PR DESCRIPTION
**Pull request by danmihaila**

### CODE REVIEW

File: hdx_hapi/endpoints/get_currency_response.py
Comment (IanHopkinson) [inline hdx_hapi/endpoints/get_currency_response.py:1]: I think this should be `get_currency.py` for consistency with other endpoints

File: hdx_hapi/endpoints/models/currency_response.py
Comment (IanHopkinson) [inline hdx_hapi/endpoints/models/currency_response.py:1]: Similarly this one should be `currency.py` for consistency

File: hdx_hapi/endpoints/get_currency_response.py
Comment (IanHopkinson) [reply hdx_hapi/endpoints/get_currency_response.py:1]: The spreadsheet asks for only `code` as a query parameter but I wonder whether this should be queryable by name too

File: tests/test_endpoints/endpoint_data.py
Comment (IanHopkinson) [inline tests/test_endpoints/endpoint_data.py:1]: Are you attempting to check for case invariance here or is "usD" a typo?

**Review by IanHopkinson — APPROVED** [review]
This looks fine - I just had a couple of queries over names.

File: hdx_hapi/endpoints/get_currency_response.py
Comment (danmihaila) [reply hdx_hapi/endpoints/get_currency_response.py:1]: i had the same question, but i just did it as it was in spreadsheet. we can review next week. ok with renaming

File: hdx_hapi/endpoints/models/currency_response.py
Comment (danmihaila) [reply hdx_hapi/endpoints/models/currency_response.py:1]: ok

File: tests/test_endpoints/endpoint_data.py
Comment (danmihaila) [reply tests/test_endpoints/endpoint_data.py:1]: initially I did it with sensitive, but i checked other endpoints and switch to insensitive. we can easy change if DS team give us feedback

**Review by alexandru-m-g — APPROVED** [review]

**Pull request merged**